docs/api/notification.md

```diff
@@ -144,6 +145,12 @@ A `Boolean` property representing whether the notification is silent.
 
 A `Boolean` property representing whether the notification has a reply action.
 
+#### `notification.urgency` _Linux_
+
+A `String` property representing the urgency level of the notification. Can be 'normal', 'critical', or 'low'.
+
+Default is 'low' - see [NotifyUrgency](https://developer.gnome.org/libnotify/unstable/NotifyNotification.html#notify-notification-set-urgency) for more information.
```

Comment:
Slightly preferable to link to the reference spec rather than the implementation docs

```suggestion
Default is 'low' - see [NotifyUrgency]( https://developer.gnome.org/notification-spec/#urgency-levels) for more information.
```